lib/rules/prefer-message-ids.js

```diff
@@ -23,7 +23,10 @@ module.exports = {
     messages: {
       messagesMissing:
         '`meta.messages` must contain at least one violation message.',
-      foundMessage: 'Use `messageId` instead of `message`.',
+      // Currently the codebase does not distinguish between `message` and `desc`.
+      // `utils.collectReportViolationAndSuggestionData` returns `message` for both.
```

Comment:
```suggestion
```

I think we can omit this comment. It's helpful context for the PR description but we probably don't need a comment for it.